sort/bubble.go

```
@@ -0,0 +1,18 @@
+// Time Complexity: O(n²) where n is the length of the slice
+// Space Complexity: O(1) as it sorts in-place
+func BubbleSort[T number](arr []T) []T {
+	n := len(arr)
+	for i := range n - 1 {
+		for j := range n - i - 1 {
+			if arr[j] > arr[j+1] {
+				arr[j], arr[j+1] = arr[j+1], arr[j]
+			}
+		}
+	}
+	return arr
```

golangci-lint (1.62.2)

return with no blank line before

(nlreturn)